tests/boto_tests/test_utils.py

```diff
@@ -9,18 +11,17 @@
 from aiobotocore import utils
 from aiobotocore._helpers import asynccontextmanager
 
+# TypeAlias (requires typing_extensions or >=3.10 to annotate)
+Response = Tuple[Union[str, object], int]
+
 
 # From class TestContainerMetadataFetcher
-def fake_aiohttp_session(
-    responses: Union[
-        List[Tuple[Union[str, object], int]], Tuple[Union[str, object], int]
-    ]
-):
+def fake_aiohttp_session(responses: list[Response] | Response):
```

Comment:
doesn't the `|` operator require 3.10+?  https://peps.python.org/pep-0604/

Reply:
The `from __future__ import annotations` enables https://peps.python.org/pep-0563/, which means the code runs (as can be seen in the tests passing) on <3.10 - but it might require your type checker environment to be running on python 3.10+. That's overwhelmingly the case though, and `from __future__ import annotations` is used very widely in my experience as it makes typehints more readable & enables features.